schemas/jsonschema/CompressedImage.json

```diff
@@ -1,7 +1,7 @@
 {
   "$comment": "Generated from CompressedImage by @foxglove/message-schemas",
   "title": "CompressedImage",
-  "description": "A compressed image",
+  "description": "A raw image",
```

Comment:
Why does this say raw? This is a compressed image. "Raw image" to me means something else:

https://en.wikipedia.org/wiki/Raw_image_format

Reply:
I guess I changed the wrong one 🤦  the other one also says compressed.